content/en/blog/2025/otel-js-sdk-2-0.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Announcing the OpenTelemetry JavaScript SDK 2.0
+linkTitle: OTel JS SDK 2.0
+date: 2025-02-21
+draft: true
```

Comment:
Oops, looked too quickly, this date should be in March not February :). Fixing that:

```suggestion
date: 2025-03-21
```

We shouldn't need the draft status.